### ISSUE TITLE
Red text looks like an error message after logging in with new user/password just fine

### ISSUE THREAD

**Issue by thunderrabbit** [brettle/meteor-accounts-deluxe#1]
If I use my meteor app as an anonymous user, everything works fine.  It's great!  

Then I log in with a new account using username and password (*) (via accounts-ui and accounts-password)

I get an apparent error message in the popup below [Change password] and [Sign out] buttons

"New login not needed. Service will be added to logged in user."

Everything seems to _work_ fine, but the red text feels like an error message that probably shouldn't be there.

(*) This error message only appears if I create an account using username and password, not Twitter Oauth, for example (via accounts-twitter)

**Comment by brettle** [comment]
Thanks for the report. This issue is caused by meteor/meteor#5110 which I just opened. I have an idea for a workaround and will try to get it implemented in the next couple days.

**Comment by thunderrabbit** [comment]
Great!  I'm subscribed to meteor/meteor#5110 now.

**Comment by brettle** [comment]
Fixed by [this commit](https://github.com/brettle/meteor-accounts-add-service/commit/1463053147194102e7393391287f36dfed4c3675). To get the fix, just do:

```
meteor update
```

in your app.

**Comment by thunderrabbit** [comment]
Perfect; it works beautifully.  Thank you!

**Comment by nicodinh** [comment]
same issue here

```
Recent versions:
  0.2.0  August 15th, 2015
  0.3.0  August 18th, 2015
  0.3.1  September 7th, 2015
  0.3.2  September 26th, 2015  installed
  0.3.3  September 26th, 2015  installed
```

![noob3](https://cloud.githubusercontent.com/assets/7486270/11031176/df594dc0-86d2-11e5-8345-472fe7360f3d.JPG)

**Comment by brettle** [comment]
What accounts UI package(s) and version(s) are you using?

**Comment by nicodinh** [comment]
```
useraccounts:core                1.12.4
useraccounts:semantic-ui         1.12.4
```

**Comment by brettle** [comment]
I just confirmed that the demo works for me with Semantic UI. It's on [the `semantic-ui` branch of the demo repo](https://github.com/brettle/meteor-accounts-deluxe-demo/tree/semantic-ui). To confirm that also works for you, please:

``` sh
$ git clone git@github.com:brettle/meteor-accounts-deluxe-demo.git
$ cd meteor-accounts-deluxe-demo
$ git checkout semantic-ui
$ meteor
```

Then visit [http://localhost:3000](http://localhost:3000), change the dropdown to `useraccounts`, and try to reproduce the problem or try to figure out what's different about your app/environment.

Hope that helps. Let me know what you find.

**Comment by nicodinh** [comment]
I added useraccounts:iron-routing package and removed my custom route.
It's works !
@brettle Thank you.

**Comment by brettle** [comment]
Glad to hear you got it working. You were probably running into brettle/meteor-accounts-add-service#4.

**Comment by aberios** [comment]
I'm getting this behaviour when using kadira:flow-router with the useraccounts:flow-routing package. Do you have a working example that uses Flow Router?

**Comment by brettle** [comment]
I don't have a working example with Flow Router. If you can point me to a repo containing a minimal app that reproduces the problem, I'll take a look.

**Comment by brettle** [comment]
I just created a working example with flow-routing and it doesn't seem to have the problem you are describing.

```
git clone --branch flow-routing https://github.com/brettle/meteor-accounts-deluxe-demo.git
cd meteor-accounts-deluxe-demo
meteor
```

**Comment by aberios** [comment]
Just tried your branch, and I'm experiencing the same behaviour.  It only occurs with the useraccounts flow, not accounts-ui.

**Comment by brettle** [comment]
My bad. I see it now. I'd forgotten that the problem only occurred when creating a password user. I'd been testing with oauth users. Anyway, I'm looking into it.

**Comment by brettle** [comment]
@aberios, I just published version 0.2.2 of `brettle:accounts-deluxe` which uses a new version of `brettle:accounts-patch-ui` (0.1.6) that should fix the problem. Let me know if it doesn't.
